akka-http-core/src/test/scala/akka/http/model/UriSpec.scala

```diff
@@ -406,7 +418,7 @@ class UriSpec extends WordSpec with Matchers {
       normalize("http://sourceforge.net/projects/uriparser/") shouldEqual "http://sourceforge.net/projects/uriparser/"
       normalize("http://sourceforge.net/project/platformdownload.php?group_id=182840") shouldEqual "http://sourceforge.net/project/platformdownload.php?group_id=182840"
       normalize("mailto:test@example.com") shouldEqual "mailto:test@example.com"
-      normalize("file:///bin/bash") shouldEqual "file:///bin/bash"
+      normalize("file:///bin/bash") shouldEqual "file:/bin/bash"
```

Comment:
Is this change correct for any scheme? Browsers seem to do the conversion in the other direction for the `file` scheme. The [normalization paragraph of RFC 3986](http://tools.ietf.org/html/rfc3986#section-6.2.3) doesn't seem to specify a general normalization rule of this kind but rather:

> Normalization should not remove delimiters when their associated component is empty unless licensed to do so by the scheme specification.